src/test/scala/com/microsoft/hyperspace/index/CreateIndexTests.scala

```diff
@@ -87,6 +85,27 @@ class CreateIndexTests extends SparkFunSuite with SparkInvolvedSuite {
     assert(exception.getMessage.contains("Index config is not applicable to dataframe schema"))
   }
 
+  test("Index creation passes with columns of different case if case-sensitivity is false.") {
+    hyperspace.createIndex(df, IndexConfig("index1", Seq("qUeRy"), Seq("ImpRS")))
+    val indexes = hyperspace.indexes.where(s"""name = "${indexConfig1.indexName}" """)
+    assert(indexes.count == 1)
+    assert(
+      indexes.head.getAs[WrappedArray[String]]("indexedColumns").head.equals("Query"),
```

Comment:
nit: `== "Query"` is easier to read, esp. when you have a long chain.

Reply:
thanks @imback82 , fixed